scripts/image_processors/noaa_pristine.sh

```diff
@@ -18,4 +18,4 @@ INPUT_WAV=$1
 OUTPUT_IMAGE=$2
 
 # produce the output image
-$WXTOIMG -o -e "pristine" "${INPUT_WAV}" "${OUTPUT_IMAGE}"
+$WXTOIMG -o -e -16 "pristine" "${INPUT_WAV}" "${OUTPUT_IMAGE}"
```

Comment:
I am curious, what's this do?

Reply:
The ```-16``` (from the manual): 

_Write 16 bit grayscale. Only PNG and PBM (PGM) formats can support 16 bit grayscale._